src/components/Reown/modals/SendNanoContractTxModal.js

```diff
@@ -47,13 +48,13 @@ export function SendNanoContractTxModal({ data, onAccept, onReject }) {
     }
 
     const methodInfo = blueprintInfo.public_methods?.[data.data.method];
-    const argEntries = methodInfo
-      ? data.data.args.map((arg, idx) => [
-          methodInfo.args[idx].name,
-          arg,
-          methodInfo.args[idx].type
-        ])
-      : data.data.args.map((arg, idx) => [t`Position ${idx}`, arg]);
+    const methodInfoArgs = get(methodInfo, 'args', []);
+    const dataArgs = get(data.data, 'args', []);
```

Comment:
Can we show an error if `methodInfoArgs.length !== dataArgs.length`? I think it doesn't make sense to continue with the request if the arguments count is different than the expected count.

Reply:
We'll handle this in the rpc-lib when we start validating blueprint arguments

I don't think we should validate it in the wallets